docs/apis-tools/migration-manuals/migrate-component-apis.md

```diff
@@ -0,0 +1,63 @@
+---
+id: migrate-component-apis
+title: Migrate Component V1 APIs
+description: "Making necessary changes to continue using Camunda's V1 component REST APIs."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This document offers an overview over necessary changes to continue working with the component REST APIs after an upgrade to Camunda 8.8 if a migration to the new [Orchestration cluster REST API](/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md) is not possible.
```

Comment:
🙃 We can add a sentence here explaining what the "components" are. Maybe just something like this:

```suggestion
This document offers an overview over necessary changes to continue working with the component (Operate, Tasklist) REST APIs after an upgrade to Camunda 8.8 if a migration to the new [Orchestration cluster REST API](/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md) is not possible.
```